pkg/config/defaults.go

```diff
@@ -46,5 +47,9 @@ func applyDefaults(cfg *Finch, deps LoadSystemDeps, mem fmemory.Memory) *Finch {
 		cfg.Rosetta = pointer.Bool(false)
 	}
 
+	if cfg.HostGatewayIP == nil {
```

Comment:
Maybe we don't have to fill host gateway IP for users by default. We can keep it nil and process it based on nil check. That behaviour is docker compliance.

Finch also has an existing field "additional_directories" that isn't filled by default.